lib/scripts/top-bar.js

```diff
@@ -0,0 +1,35 @@
+const topBar = document.querySelector('.desktop-bar');
+function renderTopBar() {
+  topBar.innerHTML = `<div class="tool-bar">
+  <img
+    class="tool-bar-item"
+    src="images/social/facebook.svg"
+    alt="facebook-icon"
+  />
+  <img
+    class="tool-bar-item"
+    src="images/social/twitter.svg"
+    alt="twitter-icon"
+  />
+  <p class="tool-bar-item">English</p>
+  <p class="tool-bar-item">My Page</p>
+  <p class="tool-bar-item last-item">Logout</p>
+</div>
+<div class="navbar">
```

Comment:
Both pages contain the same navbar and footer 🚀

Reply:
It's important that your HTML code follows proper semantics 🧐 Please change this `div` tag into a `nav` tag to improve this aspect of your project 🙌